.changeset/silver-needles-rush.md

```diff
@@ -0,0 +1,5 @@
+---
+'@aws-amplify/auth-construct-alpha': patch
+---
+
+Update frontend config to output OIDC provider names instead of just 'OIDC'.
```

Comment:
How is this changing the frontend client config. The change only seems to have an effect on the output stored in the template but not the frontend config generation.

Reply:
we were already parsing the list of socialProviders, so that logic isn't changing, just the data that is being parsed which is controlled by the construct ; this affects the frontend config socialProviders values